cmd/crdvalidator/manifests/00_service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: default
```

Comment:
I'm not sure how I feel about deploying all these resources to the default namespace, generally it's considered best practice not to deploy to the default namespace. 

An alternative is to deploy to the rukpak-system namespace that we are using currently, but we are coupling this webhook to rukpak more in that case. 

WDYT about shipping this with it's own namespace so that all the resources are self-contained in that namespace?

Reply:
I like the concept of a `crdvalidator-system` (or something in that vein) namespace. I didn't do it to begin with to make sure that was something we wanted to go with but it makes sense. In the theme of not coupling this with Rukpak I'd prefer it to be in its own - would you agree?